tests/python_tests/test_tokenizer.py

```diff
@@ -109,7 +107,12 @@ def get_chat_templates():
 @pytest.mark.precommit
 @pytest.mark.nightly
 def test_encode(model_descr, prompt):
-    model_id, path, hf_tokenizer, opt_model, ov_pipe = read_model(model_descr)
+    model_id, tmp_path = model_descr
+    opt_model, hf_tokenizer, models_path = download_and_convert_model(model_id, tmp_path)
+    ov_pipe = create_ov_pipeline(models_path=models_path,
+                                 pipeline_type=PipelineType.STATEFUL,
+                                 ov_config=get_disabled_mmap_ov_config())
```

Comment:
why do we need this `get_disabled_mmap_ov_config()` ? I would drop it

Reply:
This question should be addressed to test author.
I will remove it, but cannot provide the correct answer why these property were used originally

Reply:
I would separately create a PR (after this one is merged) and dropped this `get_disabled_mmap_ov_config` from all places.

Looks like WA for some other issue.